### DISCUSSION TITLE
Blackboard background and chalkboard

### DISCUSSION THREAD

**Discussion by hephesto** [xournalpp/xournalpp#2993] [Ideas]
I would like to suggest the inclusion of a **blackboard** (or green board) as a background and **chalkboard** for writing. 

I know of the option to include a background image but under Linux the program hangs. It would be better if you already had it as a standard. 

Thank you.

**Comment by personalizedrefrigerator** [comment]
> I know of the option to include a background image but under Linux the program hangs.

I'm not experiencing this issue with Fedora 33 / Xournalpp 1.1.0+dev. What version of Xournal++ are you using?
If you're using Xournal++ 1.1.0+dev, I suggest you [open an issue](https://github.com/xournalpp/xournalpp/issues/new/choose) for this.

> It would be better if you already had it as a standard.

This might not be quite what you want, but I use `Journal > Configure Page Template > Background Color / Background Style` to make the default page style blackboard-like.

I hope this is helpful!

**Reply by hephesto** [reply]
I am using Xournal++ 1.0.20, built on with gtklib 3.24.20, in a Ubuntu distro:

$ lsb_release -a
LSB Version:	core-11.1.0ubuntu2-noarch:printing-11.1.0ubuntu2-noarch:security-11.1.0ubuntu2-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

How about Background Style, yes, it helps me. I really appreciated your help.

Thank you very much.
